recipes-graphics/spark/spark_git.bb

```diff
@@ -0,0 +1,100 @@
+include pxcore.inc
+
+DEPENDS_append = " curl freetype util-linux libjpeg-turbo libpng pxcore-libnode giflib sqlite3 wpeframework-tools-native wpeframework wpeframework-clientlibraries"
```

Comment:
Can you please split that into multiple lines?

Reply:
done